controllers/profiles/getOtherProfile.js

```diff
@@ -48,7 +48,7 @@ module.exports = async (req, res) => {
     });
 
     const isFollowing = await sequelize.query(isFollowingQuery, {
-      replacements: {user_id_followed: other_user_id, user_id_follower: req.userId}
+      replacements: {user_id_follower: other_user_id, user_id_followed: req.userId}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the order of `user_id_followed` and `user_id_follower` in the replacements object could potentially affect the logic of the query. Please ensure that the SQL query `isFollowingQuery` is expecting the parameters in this new order. If not, it might lead to incorrect results or even a runtime error.
```diff
- replacements: {user_id_follower: other_user_id, user_id_followed: req.userId}
+ replacements: {user_id_followed: other_user_id, user_id_follower: req.userId}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->